locations/spiders/DskBank.py

```diff
@@ -0,0 +1,66 @@
+import re
+
+import scrapy
+
+from locations.categories import Categories, apply_category
+from locations.hours import DAYS_BG, OpeningHours, sanitise_day
+from locations.items import Feature
+
+
+class DSKBankBGSpider(scrapy.Spider):
+    name = "dsk_bank_bg"
+    item_attributes = {"brand": "Банка ДСК", "brand_wikidata": "Q5206146"}
+    allowed_domains = ["https://www.dskbank.bg/"]
+    start_urls = ["https://dskbank.bg/контакти/клонова-мрежа/GetOffices/"]
+
+    def parse(self, response):
+        for data in response.json():
+            item = Feature()
+            item["ref"] = data["Id"]
+            item["name"] = data["Name"]
+            item["lat"] = data["Latitude"]
+            item["lon"] = data["Longitude"]
+            item["email"] = data["Email"]
```

Comment:
I think you could replace all of this with `item = DictParser.parse(data)` after adding an import `from locations.dict_parser import DictParser`